bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/vehicle.xml

```diff
@@ -9,10 +9,10 @@
 		</supported-bridge-type-refs>
 		<label>Vehicle</label>
 		<description>A vehicle configured in your evcc instance</description>
-		<representation-property>id</representation-property>
+		<representation-property>vehicle-id</representation-property>
 		<config-description>
-			<parameter name="id" type="text">
-				<label>ID</label>
+			<parameter name="vehicle-id" type="text">
```

Comment:
`camelCase` naming convention applies here.
For reference: https://www.openhab.org/docs/developer/guidelines.html#naming-convention